### ISSUE TITLE
Sounds stop playing after playing many sequentially

### ISSUE THREAD

**Issue by Lucrecious** [libgdx/libgdx#4296]
After playing some sounds quickly sequentially (as quickly as I can tap keys on my keyboard), they stop playing (after like 4 - 5 seconds).

I'm on the 1.9.2 libgdx.

There's no stacktrace, just that I get the -1 on sound ids after playing ANY sound after the bug.

- [X] Windows
- [X] MacOS

The ones not listed I'm not sure about, I haven't able to test them there.

I'm thinking the sounds might be being disposed of because in the same game instance, if I reload the sounds, they all play fine.

Any ideas?

**Comment by xoppa** [comment]
https://github.com/libgdx/libgdx/wiki/More-Info

**Comment by Lucrecious** [comment]
Thanks, but what I wrote down was pretty much all I knew.

I didn't want to release my project executable or source code because I wanted to keep it private for the moment being. I was hoping just to get some general advice, heh.

After some research and looking around, I came up with a theory that sounds stop playing after a certain number. I tested out that theory, and it turned out to be my problem. I had a bunch of sounds at 0 volume playing (so I couldn't hear them) and after the sound number limit, all other sounds weren't allowed to play. Resulting in my dumb bug.

One line of code fixed it, heh.

Could someone tell me how to access the max concurrent sounds allowed? Or tell me what it is if it's a hardcoded value?

Thanks for the comment, I'll make sure to be more detailed next time :)

tl;dr for anyone who might be having similar sound issues, make sure you're under the sound number limit.

**Comment by intrigus** [comment]
That should be the sound number limit:
https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglApplicationConfiguration.java#L74
